blog/2019-04-25-pulsar-debezium-tutorial.adoc

```diff
@@ -0,0 +1,242 @@
+
+Start a MySQL CLI docker connector,  and you can make changes to the “products” table in MySQL server.
+
+[source,bash]
+----
+$docker run -it --rm --name mysqlterm --link mysql --rm mysql:5.7 sh -c 'exec mysql -h"$MYSQL_PORT_3306_TCP_ADDR" -P"$MYSQL_PORT_3306_TCP_PORT" -uroot -p"$MYSQL_ENV_MYSQL_ROOT_PASSWORD"'
+----
+
+After running the command, MySQL CLI is displayed, and you can change the names of the two items in the “products” table.
+
+[source,bash]
+----
+mysql> use inventory;
+mysql> show tables;
+mysql> SELECT * FROM  products ;
+mysql> UPDATE products SET name='1111111111' WHERE id=101;
+mysql> UPDATE products SET name='1111111111' WHERE id=107;
+----
+
+++++
+<div class="imageblock centered-image">
+    <img src="/images/pulsar_tutorial/pulsar-mysql-4.png" style="max-width:100%; margin-bottom:20px; margin-top:20px;" class="responsive-image" alt="mysql updates">
+</div>
+++++
+
+In the terminal where you consume products topic, you find that two changes have been added.
+
+++++
+<div class="imageblock centered-image">
+    <img src="/images/pulsar_tutorial/pulsar-mysql-5.png" style="max-width:100%; margin-bottom:20px; margin-top:20px;" class="responsive-image" alt="table topic stores mysql updates">
+</div>
+++++
+
+In the terminal where you consume the offset topic, you find that two offsets have been added.
+
+++++
+<div class="imageblock centered-image">
+    <img src="/images/pulsar_tutorial/pulsar-mysql-6.png" style="max-width:100%; margin-bottom:20px; margin-top:20px;" class="responsive-image" alt="offset topic get updated">
+</div>
+++++
+
+In the terminal where you local-run the connector, you find two more records have been processed.
+
+++++
+<div class="imageblock centered-image">
+    <img src="/images/pulsar_tutorial/pulsar-mysql-7.png" style="max-width:100%; margin-bottom:20px; margin-top:20px;" class="responsive-image" alt="table topic get more records">
+</div>
+++++
+
+### Step 6: Clean up. 
+
+Use “Ctrl +C” to close terminals. Use “docker ps” and “docker kill” to stop MySQL related docker instances. 
+
+[source,bash]
+----
+mysql> quit
+
+$ docker ps
+CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS              PORTS                               NAMES
+84d66c2f591d        debezium/example-mysql:0.8   "docker-entrypoint.s…"   About an hour ago   Up About an hour    0.0.0.0:3306->3306/tcp, 33060/tcp   mysql
+
+$ docker kill 84d66c2f591d
+----
+
+To delete Pulsar data, delete data directory in the Pulsar binary directory.
+
+[source,bash]
+----
+$ pwd
+/Users/jia/ws/releases/apache-pulsar-2.3.0
+
+$ rm -rf data 
+----
+
+## Conclusion
+Pulsar Debezium Connector is a user-friendly project for change data capture. After walking through this tutorial, you learn how to store MySQL data changes using Pulsar source connector. We are improving Pulsar Debezium Connector continuously, it would be much easier to use after Pulsar 2.4.0 release.
```

Comment:
See above; perhaps: Pulsar IO allows to run the Debezium connectors for change data capture, streaming data changes from different databases into Apache Pulsar. In this tutorial you've learned how to capture data changes in a MySQL database and propagate them to Pulsar. ...